### PR TITLE
Fixed solo host banning

### PR DESCRIPTION
**Pull request by TideSofDarK**
When only host bans are enabled, banning phase should end when host is done banning.
Currently, the game waits like 1.5 minutes after u have clicked finished, when its only host banning.

### CODE REVIEW

**Comment by darklordabc** [comment]
Tested this in a 1 player match. And it worked. When the host finished bannings, and other players had no more bans left, the ban round ended as it should off.

**Comment by darklordabc** [comment]
LGTM

File: src/scripts/vscripts/pregame.lua [hunk 1]
```diff
@@ -3962,6 +3966,20 @@ function Pregame:getActivePlayers()
     return total
 end
 
+-- Get host player
+function Pregame:getHostPlayer()
+	for i=0,24 do
```
Comment (Myrl) [inline src/scripts/vscripts/pregame.lua]: Aside from this, everything seems to be fine. Use GetPlayerCount instead and remove IsValidPlayer.

**Comment by darklordabc** [comment]
LGTM

**Comment by TideSofDarK** [comment]
LGTM

**Comment by Myrl** [comment]
LGTM.

**Comment by darklordabc** [comment]
merged

**Pull request closed without merging**